src/java/org/apache/cassandra/db/compaction/CompactionStrategyOptions.java

```diff
@@ -107,7 +111,7 @@ public CompactionStrategyOptions(Class<? extends CompactionStrategy> klass, Map<
         tombstoneCompactionInterval = Long.parseLong(getOption(TOMBSTONE_COMPACTION_INTERVAL_OPTION, useDefault, DEFAULT_TOMBSTONE_COMPACTION_INTERVAL));
         uncheckedTombstoneCompaction = Boolean.parseBoolean(getOption(UNCHECKED_TOMBSTONE_COMPACTION_OPTION, useDefault, DEFAULT_UNCHECKED_TOMBSTONE_COMPACTION_OPTION));
         logAll = Boolean.parseBoolean(getOption(LOG_ALL_OPTION, useDefault, DEFAULT_LOG_ALL_OPTION));
```

Comment:
I would not track the old setting. Instead, `LOG_ALL_OPTION`, if defined, should set `logType` to `ALL`/`NONE` (respectively for `true`/`false`).